snap/snapcraft.yaml

```
@@ -33,21 +35,18 @@ plugs:
 apps:
   device-mqtt:
     adapter: full
-    command: bin/device-mqtt -configDir $SNAP_DATA/config/device-mqtt -profile res --registry $CONSUL_ADDR
+    command: bin/device-mqtt --configDir $SNAP_DATA/config/device-mqtt --configProvider --registry -profile res
```

Are these equivalent?
```suggestion
    command: bin/device-mqtt --configDir $SNAP_DATA/config/device-mqtt/res --configProvider --registry
```

---

Yes, they are equivalent.